common/benchmark_utils.py

```diff
@@ -163,3 +163,20 @@ def get_language(benchmark):
     """Returns the prorgamming language the benchmark was written in."""
     config = benchmark_config.get_config(benchmark)
     return config.get('language', 'c++')
+
+
+def are_benchmarks_mixed(benchmarks):
+    """"Returns True if benchmarks list
```

Comment:
Please fix the formatting of this docstring.

Reply:
Just realized there's an extra apostrophe there.

Is there something else to fix, like the line break? I did the line break due to a lint error

Reply:
Right I mean the line break. It's formatted a bit weirdly. You should try to put as many chars as possible on a docstring line before breaking if youre continuing the same sentence. Only "benchmarks" needs to be on the second.
My editor has a shortcut for this btw: https://www.gnu.org/software/emacs/manual/html_node/emacs/Fill-Commands.html (fill-paragraph)
You probably shouldn't switch to my editor, but yours probably has something like this.
Also please end all sentences with a period.

Reply:
Thanks, I am using Rewrap on vscode and its working now